app/apps/visits/models.py

```diff
@@ -1,30 +1,71 @@
 from apps.itinerary.models import ItineraryItem
 from apps.users.models import User
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
 
 
 class Visit(models.Model):
     """ Captures data of a visit """
 
-    itinerary_item = models.ForeignKey(
-        ItineraryItem, on_delete=models.CASCADE, null=False, related_name="visits"
+    SITUATION_NOBODY_PRESENT = "nobody_present"
+    SITUATION_NO_COOPERATION = "no_cooperation"
+    SITUATION_ACCESS_GRANTED = "access_granted"
+
+    SITUATIONS = (
+        (SITUATION_NOBODY_PRESENT, "Niemand aanwezig"),
+        (SITUATION_NO_COOPERATION, "Geen medewerking"),
+        (SITUATION_ACCESS_GRANTED, "Toegang verleend"),
+    )
+
+    OBSERVATION_MALFUNCTIONING_DOORBEL = "malfunctioning_doorbell"
+    OBSERVATION_VIDEO_CALL = "video_call"
+    OBSERVATION_HOTEL_FURNISHED = "hotel_furnished"
+    OBSERVATION_VACANT = "vacant"
+    OBSERVATION_LIKELY_INHABITED = "likely_inhabited"
+
+    OBSERVATIONS = (
+        (OBSERVATION_MALFUNCTIONING_DOORBEL, "Bel functioneert niet"),
+        (OBSERVATION_VIDEO_CALL, "Contact via videobel"),
+        (OBSERVATION_HOTEL_FURNISHED, "Hotelmatig ingericht"),
+        (OBSERVATION_VACANT, "Leegstaand"),
+        (OBSERVATION_LIKELY_INHABITED, "Vermoedelijk bewoond"),
+    )
+
+    SUGGEST_VISIT_WEEKEND = "weekend"
+    SUGGEST_VISIT_DAYTIME = "daytime"
+    SUGGEST_VISIT_EVENING = "evening"
+    SUGGEST_VISIT_UNKNOWN = "unknown"
+
+    SUGGEST_NEXT_VISIT = (
+        (SUGGEST_VISIT_WEEKEND, "Weekend"),
+        (SUGGEST_VISIT_DAYTIME, "Overdag"),
+        (SUGGEST_VISIT_EVENING, "'s Avonds"),
+        (SUGGEST_VISIT_UNKNOWN, "Onbekend"),
     )
-    author = models.ForeignKey(
-        to=User, null=False, blank=False, on_delete=models.CASCADE
+
+    status = models.CharField(max_length=50, choices=SITUATIONS, null=True, blank=True)
+    observations = ArrayField(
```

Comment:
Zelf heb ik ArrayField nog nooit gebruikt, maar goed om te weten dat dit kan als je PostgreSQL gebruikt met Django. 👍

Reply:
Werkt fijn en precies wat je wilt ipv een one to many relatie met een simpel choices model